javascript/exosecurity/src/exosecurity.js

```diff
@@ -0,0 +1,58 @@
+const uuid = require('uuid')
+const { EventEmitter } = require('events')
+const ExoRelay = require('../../exorelay')
+const { delegate, delegateEvent } = require('rails-delegate')
+
+class Exosecurity extends EventEmitter {
+  constructor({ exocomHost, exocomPort, role, messageAuthorizer }) {
+    super()
+    this.exoRelay = new ExoRelay({
+      exocomHost,
+      exocomPort,
+      role,
+    })
+    this.handlers = {
+      'authorize message': (message, { reply }, activityId) =>
+        messageAuthorizer(message, this.buildRequester(activityId), result => {
+          if (result) {
+            return reply('message authorized')
+          }
+          return reply('message unauthorized')
```

Comment:
thoughts on drying this up by defining a messageName variable? Then we would only have one use of `reply`